src/packages/client/src/__tests__/types/json-filtering-mysql/index.test-d.ts

```diff
@@ -0,0 +1,34 @@
+import { PrismaClient } from '.'
+import { expectError } from 'tsd'
+
+// tslint:disable
```

Comment:
Why is that needed? 👀

Reply:
No idea was copy/pasted from another test